backend/lambdas/letter-processor/src/validation.ts

```
@@ -0,0 +1,82 @@
+  if (typeof value !== 'string') return false
+  if (!DATE_PATTERN.test(value)) return false
+
+  // Verify it's a real date
+  const date = new Date(value)
+  return !isNaN(date.getTime())
+}
+
+/**
+ * Safely convert unknown value to string or null
+ */
+function toString(value: unknown): string | null {
```

Biome (2.1.2)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)